docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -196,6 +197,21 @@ make up
 NOTE: When Lisk Service is started again after a database reset, then the process to reindex all the data is initiated.
 This can be quite time-consuming when the chain is long, spanning over hours.
 
+// == MySQL snapshot
+== Taking Backups
+
+If you wish to take a backup of your docker installation, the details can be found here on the xref:{url_back_up}[Backups] page.
+
```

Comment:
Please remove the commented out content!

Reply:
Done! 👍